velox/vector/arrow/Bridge.h

```diff
@@ -36,6 +36,7 @@ struct ArrowOptions {
   bool flattenDictionary{false};
   bool flattenConstant{false};
   TimestampUnit timestampUnit = TimestampUnit::kNano;
+  std::optional<std::string> timestampTimeZone = {"UTC"};
```

Comment:
Since this object may end up being constructed often during arrow conversions, can we make it cheap and by default assume the optional is std::nullopt, instead of making it allocate the UTC string every time?

Reply:
@pedroerp Good suggestion. Update the default value to `std::nullopt`.